File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Projects/saveprojectdatatoexcel.rb
module Asposetasksjava
  module SaveProjectDataToExcel
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        # Save the Project as XLSX
        project.save(data_dir + "Project.XLSX", Rjb::import('com.aspose.tasks.SaveFileFormat').XLSX)

        puts "Saved project data to Excel, please check the output file"

    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithProjects/SaveProjectDataToExcel.php
<?php
namespace Aspose\Tasks\WorkingWithProjects;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\SaveFileFormat as SaveFileFormat;

class SaveProjectDataToExcel{

    public static function run($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        # Save the Project as XLSX
        $saveFileFormat=new SaveFileFormat();
        $project->save($dataDir . "Project.XLSX", $saveFileFormat->XLSX);

        print "Saved project data to Excel, please check the output file".PHP_EOL;

    }
}
?>